pom.xml

```diff
@@ -425,7 +425,7 @@
     <dependency>
       <groupId>org.apache.iceberg</groupId>
       <artifactId>iceberg-spark-runtime-3.3_2.12</artifactId>
-      <version>1.4.2-affirm0</version>
+      <version>1.5.0</version>
```

Comment:
oh nice, do we not need the fork anymore? Didn't know we had one.

Reply:
This was to fix the `DELETE FROM {table}` bugs we previously encountered. Nice to not rely on the fork anymore!